src/image.ts

```diff
@@ -42,6 +42,11 @@ function src(salt: string, input: string, width: number): string {
     return `${imageResizer}/${service}${url.pathname}?${params.toString()}`;
 }
 
+const srcsetWithWidths = (widths: number[]) => (url: string, salt: string): string =>
```

Comment:
These functions have changed in the liveblog PR 😅

I noticed we still have them duplicated in the codebase.

Reply:
Yeah, that's my fault 😬. Fixing that is on my list, but I've held off for a bit to avoid conflicts with #231 and #244.